### PR TITLE
FIX: Bonferroni reject mask bug

### LINKED ISSUE #2227: BUG/API issue with multi_comp.bonferroni_correction
Do people agree that this is a bug, at least a very unexpected output:
https://github.com/mne-tools/mne-python/blob/master/mne/stats/multi_comp.py#L101

`pval_corrected` should instead be used for creating the output mask.

@agramfort @Eric89GXL @mainakjas

### PR DESCRIPTION
**Pull request by dengemann**
closes #2227.

### CODE REVIEW

File: mne/stats/tests/test_multi_comp.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import numpy as np
-from numpy.testing import assert_almost_equal, assert_allclose, assert_raises
+from numpy.testing import (
+    assert_almost_equal, assert_allclose, assert_raises, assert_array_equal)
```
Comment (mainakjas) [inline mne/stats/tests/test_multi_comp.py]: where do you use these?

**Comment by mainakjas** [comment]
lgtm +1 for merge when travis finishes

File: mne/stats/tests/test_multi_comp.py (hunk 1 above)
Comment (dengemann) [inline mne/stats/tests/test_multi_comp.py]: optimized pep8

**Comment by mainakjas** [comment]
how does this fix affect the examples?

**Comment by dengemann** [comment]
> how does this fix affect the examples?

I think we don't use it anywhere.

**Comment by dengemann** [comment]
but maybe we do ...

**Comment by mainakjas** [comment]
yeah, we do use it in two examples I think. Can you post the before and after plots?

**Comment by dengemann** [comment]
I post after. for before see mne webpage.

**Comment by mainakjas** [comment]
okay

**Comment by agramfort** [comment]
LGTM Thanks !

**Comment by dengemann** [comment]
So here it does not make any difference, the bonferroni is just illustrated but not used, the actual line is computed by hand.
https://www.dropbox.com/s/vvpz75k3gnrmez4/Screenshot%202015-06-21%2019.59.46.png?dl=0

**Comment by dengemann** [comment]
in the second exmaple the reject output is not used.

**Comment by dengemann** [comment]
so we're fine.

**Comment by agramfort** [comment]
thanks !

**Comment by mainakjas** [comment]
cool thanks @dengemann !

**Pull request merged**